design/design-doc-upbound-ui-spec.md

```diff
@@ -143,6 +143,7 @@ initialConfig:
     Property | Description
     --- | ---
     `controlType` | **singleSelect**
+    `default` | An array containing a string to set as the selected value on render in the UI. This value should also appear in the `enum` property.
```

Comment:
Is it really supposed to be an array?

Reply:
yes, it is the same for both single and multiple select